src/navigations/TabNavigation/TabNavigation.tsx

```diff
@@ -12,16 +12,35 @@ import HomeScreenV2 from '../../screens/HomeScreen/HomeScreenV2';
 import { useSelector } from 'react-redux';
 import { AllTaskScreenStack } from '../../screens/Stacks/AllStack';
 import { scale } from '../../utils/utils';
+import CalendarInviteScreen from '../../screens/Calendar/CalendarInviteScreen';
 
 const tab = createBottomTabNavigator();
 
+const renderLabel = ({ focused }) => {
```

Comment:
`renderLabel` sounds like a generic reusable function, but it seems to only be rendering label _only_ for calendar. Is this intentional?

Reply:
yeah, need to change for other tabs too

Reply:
changed